riscv-c-api.md

```diff
@@ -58,19 +58,33 @@ For example:
 - F-extension v2.2 will define `__riscv_f` as `2002000`.
 - B-extension v0.92 will define `__riscv_b` as `92000`.
 
-| Name                    | Value        | When defined                  |
-| ----------------------- | ------------ | ----------------------------- |
-| __riscv_arch_test       | 1            | Defined if compiler support new architecture extension test macro. |
-| __riscv_i               | Arch Version | `I` extension is available.   |
-| __riscv_e               | Arch Version | `E` extension is available.   |
-| __riscv_m               | Arch Version | `M` extension is available.   |
-| __riscv_a               | Arch Version | `A` extension is available.   |
-| __riscv_f               | Arch Version | `F` extension is available.   |
-| __riscv_d               | Arch Version | `D` extension is available.   |
-| __riscv_c               | Arch Version | `C` extension is available.   |
-| __riscv_b               | Arch Version | `B` extension is available.   |
-| __riscv_v               | Arch Version | `V` extension is available.   |
-| __riscv_zfh             | Arch Version | `Zfh` extension is available. |
+
+| Name                    | Value        | When defined                                                 |
+| ----------------------- | ------------ | ------------------------------------------------------------ |
+| __riscv_arch_test       | 1            | Compiler supports architecture extension test macros.        |
+| __riscv_i               | Arch Version | `I` extension is available.                                  |
+| __riscv_e               | Arch Version | `E` extension is available.                                  |
+| __riscv_m               | Arch Version | `M` extension is available.                                  |
+| __riscv_a               | Arch Version | `A` extension is available.                                  |
+| __riscv_f               | Arch Version | `F` extension is available.                                  |
+| __riscv_d               | Arch Version | `D` extension is available.                                  |
+| __riscv_c               | Arch Version | `C` extension is available.                                  |
+| __riscv_b               | Arch Version | `B` extension is available.                                  |
+| __riscv_v               | Arch Version | `V` extension is available.                                  |
+| __riscv_zfh             | Arch Version | `Zfh` extension is available.                                |
+| __riscv_zk              | Arch Version | `Zbkb` `Zbkc` `Zbkx` `Zkne` `Zknd` `Zknh` are all available. |
```

Comment:
Should Zk include Zkr and Zkt?

Reply:
Well, perhaps it should, but Zk automatically implies Zkt, as do all other cryptographic extensions -- like Zkbk alone. Zkr does not have any impact on the compiler. All cryptographic extensions are Zkt.

Reply:
Why is Zkt only explicitly listed as implied by Zk in the spec?

Reply:
You're right. Constant time execution is not implied by other extensions to anything but those specific instructions themselves; no Zkt. Zkt in Zk implies it for all the instructions in the list. https://github.com/mjosaarinen/riscv-c-api-doc/commit/27183d099a72560ea865b5443ae1333919324df4